packages/model-viewer/src/three-components/ModelScene.ts

```diff
@@ -542,6 +542,7 @@ export class ModelScene extends Scene {
       const side =
           (this.element as any).arPlacement === 'wall' ? 'back' : 'bottom';
```

Comment:
Nit: The names 'back' and 'bottom' are a bit ambiguous, bottom of what? Why not just support 'wall' and 'floor' and the pass .arPlacement value directly?

Reply:
They refer to the bounding box; the type is from the  Shadow class for which  these names seemed to make more sense, though hard to say. The real issue is that I haven't found a good way to type the attributes, so I used this instead. We should probably come back to this though.